dbms/src/Flash/tests/gtest_interpreter.out

```diff
@@ -0,0 +1,463 @@
+~func_name: SingleQueryBlock
```

Comment:
Rename to `test_suite_name` maybe better

Reply:
ok, renamed